test/browser/index.js

```diff
@@ -22,6 +22,14 @@ const fileContent = new Buffer(
   '</svg>'
 );
 
+const fileName2 = 'foobaz.svg';
```

Comment:
For some reason im getting connection timeouts to my farmers when trying to upload the same file in succession.

Even with a deleteFile call between.